gql/utils_codegen.py

```diff
@@ -30,6 +41,13 @@ def write_lines(self, lines):
         for line in lines:
             self.lines.append(self.indent_string + line)
 
+    def block(self):
+        return self.Block(self)
+
+    def write_block(self, block_header: str, *args, **kwargs):
```

Comment:
I'm also not happy about this name either.

Reply:
`begin_block` ?

Reply:
@aviv-ebates maybe `begin_block` ?